aten/src/ATen/cpu/vml.h

```diff
@@ -39,13 +39,10 @@
 // https://bugs.launchpad.net/ubuntu/+source/glibc/+bug/1663280. Calling zeroall
 // when using AVX/AVX2 code resolves this.
 #if defined(__AVX__) && defined(__GLIBC__) && __GLIBC_MINOR__ == 23
-#define DL_RUNTIME_BUG(op, type)                              \
-  using value_t = typename at::native::ztype<type>::value_t;  \
-  volatile value_t x = (value_t)(1);                          \
-  x = std::op(x);                                             \
-  _mm256_zeroall();
+// Avoid AVX/SSE transition overhead
+#define DL_RUNTIME_BUG _mm256_zeroall();
```

Comment:
just out of curiosity, why did this change?

Reply:
for glic=2.23, **x = std::op(x);** x should have a volatile type, but for BFloat16, such as **c10::BFloat16 acos(c10::BFloat16 a) { return std::acos(float(a));}**, the parameter is not a volatile type.

Reply:
But you are changing it here not only for BFLoat16, but for all types?

Reply:
Please don't touch this. If you need something for a one-off type, please specialize the code.

Reply:
@cpuhrsch , changed as your suggestion. Thanks!